tests/test_bridge_domain_vni.rb

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/bridge_domain_vni'
+
+include Cisco
+
+# TestBridgeDomainVNI - Minitest for bridge domain class.
+class TestBridgeDomainVNI < CiscoTestCase
+  @skip_unless_supported = 'bridge_domain_vni'
+  @@cleaned = false # rubocop:disable Style/ClassVars
+
+  def cleanup
+    remove_all_vlans
+    remove_all_bridge_domains
+    BridgeDomainVNI.rangebds.each do |_bd, obj|
+      obj.destroy
+    end
+  end
+
+  def setup
+    super
+    cleanup unless @@cleaned
+    @@cleaned = true # rubocop:disable Style/ClassVars
+  end
+
+  def teardown
+    cleanup unless @@cleaned
+    super
+    cleanup
+  end
+
+  def test_single_bd_member_vni
+    mt_full_interface?
+    bd = BridgeDomainVNI.new('100')
+    assert_equal(bd.default_member_vni, bd.member_vni,
+                 'Error: Bridge-Domain is mapped to different vnis')
+
+    vni = '6000'
+    bd.member_vni = vni
+    assert_equal(vni, bd.member_vni,
+                 'Error: Bridge-Domain is mapped to different vnis')
+
+    bd.member_vni = ''
+    assert_equal(bd.default_member_vni, bd.member_vni,
+                 'Error: Bridge-Domain is mapped to different vnis')
+
+    bd.destroy
+  end
+
+  def test_multiple_bd_member_vni
+    mt_full_interface?
+    bd = BridgeDomainVNI.new('100-110,150,170-171')
+    assert_equal(bd.default_member_vni, bd.member_vni,
+                 'Error: Bridge-Domain is mapped to different vnis')
+
+    vni = '6000-6010,6050,5070-5071'
```

Comment:
Nit: You could add some whitespace just to ensure that your whitespace cleanup is working.